apps/yapms/src/lib/components/modals/authmodal/LogoutForm.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import { PocketBaseStore } from "$lib/stores/PocketBase";
+</script>
+
+<div class="flex flex-col gap-2">
+  <h3>Welcome {$PocketBaseStore.authStore.model?.email}</h3>
```

Comment:
I think this should be the user's username rather than email.

Reply:
Agreed, I'll change that.